.circleci/config.yml

```diff
@@ -422,8 +438,10 @@ workflows:
           requires:
             - x64-ubuntu-xenial
             - x64-ubuntu-bionic
+            - x64-ubuntu-focal
             - aarch64-ubuntu-xenial
             - aarch64-ubuntu-bionic
+            - aarch64-ubuntu-focal
             - x64-debian-bullseye
             - aarch64-debian-bullseye
             - x64-debian-buster
```

Comment:
It's curious that `debian` jobs are ordered by version first and architecture second whereas everything else does the opposite.

Can you please also align `debian` entries with the rest?

Reply:
I've re-ordered them by oldest to newest version and grouped them first `x64` followed by its `aarch64` twin.